app/master/build.py

```diff
@@ -87,7 +101,7 @@ def prepare(self, subjobs, project_type, job_config):
 
         self._max_executors = job_config.max_executors
         self._max_executors_per_slave = job_config.max_executors_per_slave
-        self._timing_file_path = project_type.timing_file_path(job_config.name)
+        self._timing_file_path = self.project_type.timing_file_path(job_config.name)
```

Comment:
True.  I may need to rethink this.  There are also tests that depend on being able to create a mock project_type and pass it in via this function